cmd/kube-scheduler/app/options/options_test.go

```diff
@@ -134,31 +134,6 @@ leaderElection:
 		t.Fatal(err)
 	}
 
-	v1alpha1Config := filepath.Join(tmpDir, "kubeconfig_v1alpha1.yaml")
```

Comment:
Could you add unit tests for `Deprecated.AlgorithmProvider` and `Deprecated.PolicyConfigFile`?

Reply:
Actually, not here, I'll leave a comment somewhere else

Reply:
Sure, I will follow up if the origin assignee does not continue the work.